cpp/src/io/orc/reader_impl.cu

```diff
@@ -414,20 +450,26 @@ rmm::device_buffer reader::impl::decompress_stripe_data(
         break;
       default: CUDF_EXPECTS(false, "Unexpected decompression dispatch"); break;
     }
+    decompress_check(inflate_out.data(), num_compressed_blocks, stream, any_page_failure.device_ptr());
```

Comment:
This will fail style check. To fix this, run your code through clang format using the `cudf/cpp/.clang_format` file.

Reply:
what clang-format version should I use?

Reply:
should be 11.0.0

Reply:
Formatted.